File: app/src/main/java/com/zivapp/notes/util/UtilConverter.java
package com.zivapp.notes.util;

import java.text.DecimalFormat;

public class UtilConverter {

    public static String customStringFormat(double value) {
        DecimalFormat myFormatter = new DecimalFormat("###,###.##");
        return myFormatter.format(value);
    }

    public static String customStringFormat(int value) {
        DecimalFormat myFormatter = new DecimalFormat("###,###.##");
        return myFormatter.format(value);
    }
}

File: Kotlin/app/src/main/java/com/zivapp/countandnote/util/UtilConverter.kt
package com.zivapp.countandnote.util

import java.text.DecimalFormat

class UtilConverter {
    companion object {
        fun customStringFormat(value: Double): String {
            val myFormatter = DecimalFormat("###,###.##")
            return myFormatter.format(value)
        }

        fun customStringFormat(value: Int): String {
            val myFormatter = DecimalFormat("###,###.##")
            return myFormatter.format(value)
        }
    }
}